paper/paper.bib

```diff
@@ -0,0 +1,74 @@
+@Article{Jones1998,
+author="Jones, Donald R.
+and Schonlau, Matthias
+and Welch, William J.",
+title="Efficient Global Optimization of Expensive Black-Box Functions",
+journal="Journal of Global Optimization",
+year="1998",
+volume="13",
+number="4",
+pages="455--492",
+issn="1573-2916",
+doi="10.1023/A:1008306431147",
+url="http://dx.doi.org/10.1023/A:1008306431147"
+}
+@article{scikit-learn,
+ title={Scikit-learn: Machine Learning in {P}ython},
+ author={Pedregosa, F. and Varoquaux, G. and Gramfort, A. and Michel, V.
+         and Thirion, B. and Grisel, O. and Blondel, M. and Prettenhofer, P.
+         and Weiss, R. and Dubourg, V. and Vanderplas, J. and Passos, A. and
+         Cournapeau, D. and Brucher, M. and Perrot, M. and Duchesnay, E.},
+ journal={Journal of Machine Learning Research},
+ volume={12},
+ pages={2825--2830},
+ year={2011}
+}
+@Misc{gpy2014,
+  author =   {{GPy}},
+  title =    {{GPy}: A Gaussian process framework in python},
+  howpublished = {\url{http://github.com/SheffieldML/GPy}},
+  year = {2012}
+}
+@inproceedings{hyperopt,
+    Publisher = {JMLR Workshop and Conference Proceedings},
+    Author = {Daniel Yamins and David Tax and James S. Bergstra},
+    Url = {http://jmlr.csail.mit.edu/proceedings/papers/v28/bergstra13.pdf},
+    Title = {Making a Science of Model Search: Hyperparameter Optimization in Hundreds of Dimensions for Vision Architectures},
+    Number = {1},
+    Volume = {28},
+    Editor = {Sanjoy Dasgupta and David Mcallester},
+    Year = {2013},
+    Booktitle = {Proceedings of the 30th International Conference on Machine Learning (ICML-13)},
+    Pages = {115-123}
+   }
+ @misc{msmbuilder,
+  title={msmbuilder: MSMBuilder 3.4},
+  url={http://dx.doi.org/10.5281/zenodo.48545},
+  DOI={10.5281/zenodo.48545},
+  publisher={Zenodo},
+  author={Robert T. McGibbon and
+          Matthew Harrigan and
+          Bharath Ramsundar and
+          Kyle Beauchamp and
+          Muneeb Sultan and
+          Christian Schwantes and
+          Carlos Hernández and
+          Peter Eastman and
+          Brooke Husic and
+          Patrick Riley and et al.},
+  year={2016}}
+@misc{osprey_archive,
+  author       = {Robert T. McGibbon and
+                  Carlos Hernández and
+                  Matthew Harrigan and
+                  Steven Kearnes and
+                  Muneeb Sultan and
```

Comment:
The word "functionals" in the first paragraph isn't right.

Sent from my iPhone

> On Jul 1, 2016, at 3:39 PM, Carlos Hernández <notifications@github.com> wrote:
> 
> In paper/paper.bib:
> 
> > +          Matthew Harrigan and
> > +          Bharath Ramsundar and
> > +          Kyle Beauchamp and
> > +          Muneeb Sultan and
> > +          Christian Schwantes and
> > +          Carlos Hernández and
> > +          Peter Eastman and
> > +          Brooke Husic and
> > +          Patrick Riley and et al.},
> > +  year={2016}}
> > +@misc{osprey_archive,
> > +  author       = {Robert T. McGibbon and
> > +                  Carlos Hernández and
> > +                  Matthew Harrigan and
> > +                  Steven Kearnes and
> > +                  Muneeb Sultan and
> Sorry! I can do it!
> 
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>